src/css/styles.scss

```diff
@@ -0,0 +1,87 @@
+// Introduction
+// Welcome to the CSS file! This is subjectively the most exciting code
+// of the website. Here we bring together the visual style of the website.
+//
+// Our CSS is organized using Harry Roberts' (csswizardry.com) Inverted
+// Triangle CSS (ITCSS) organizational approach. This method is mixed with
+// Block Element Modifier (BEM) naming convention for class names throughout
+// the stylesheet.
+//
+// Table of Contents:
```

Comment:
Oh wait, I forgot about the failed lint check. I think we may want to install `stylelint-config-standard-scss` and add that to the `extends` field in the stylelint config. If I do that, the only thing it complains about are the empty comments on lines 4 and 9 in this file, which can be replaced with line breaks.